styles/stylesheet.css

```diff
@@ -216,23 +351,184 @@ footer-portfolio {
     justify-content: space-between;
     align-content: center;
     flex-wrap: wrap;
-
 }
 
 .social-img {
     margin-right: 40px;
     width: 3rem;
     line-height: 3rem;
     text-align: center;
-    /* background-color: white; */
 }
 
 i {
     font-size: 2rem;
     color: white;
 }
 
+/* Footer formatting */
 
 footer p {
     color: white;
+}
+
+footer a {
+    text-decoration: none;
+}
+
+/* Section on each page for blue logo and hightlight info */
+.highlight-home {
+    display: grid;
+    grid-template-columns: auto auto;
+    grid-template-rows: auto;
+    grid-gap: 10px;
+    margin-right: 30px;
+    margin-top: 30px;
+}
+
+.highlight-home img {
+    height: 60%;
+    width: 60%;
+    padding-left: 60px;
+    padding-top: 20px;
+}
+
+
+.highlight p {
+    color: #03a3df;
+    text-transform: uppercase;
+    font-weight: bold;
+}
+
+/* Grid and Grid formatting */
+.border-top {
+    border-top: 1.5px double #444444;
+}
+
+.border-left {
+    border-left: 1.5px double #444444;
+}
+
+.border-right {
+    border-right: 1.5px double #444444;
+}
+
+.border-bottom {
+    border-bottom: 1.5px double #444444;
+}
+
+.border {
+    border: 1.5px double #444444;
+}
+
+.color {
+    color: #03a3df;
+    font-size: 1.25rem;
+}
+
+.color2 {
+    color: rgb(26, 104, 221);
+}
+
+.bold {
+    font-weight: bold;
+}
+
```

Comment:
Nice job on the naming conventions. All these classes can get tricky, but you seem to be consistent.